cfgen/cfgen

```diff
@@ -469,11 +474,11 @@ def validate_cluster_desc(desc: Dict[str, Any]) -> None:
         validate_profiles_desc(desc['profiles'], desc['pools'])
     if 'fdmi_filters' in desc and desc['fdmi_filters'] is not None:
         validate_fdmi_filters_desc(desc['fdmi_filters'], desc['nodes'])
-    logging.debug('Validations passed')
+    LOG.debug('Validations passed')
```

Comment:
I think this is good info to understand the various validation stages performed.

Reply:
Fixed.